src/Grid/grid.jl

```diff
@@ -107,6 +108,9 @@ function faces(c::AbstractCell{RefLine})
     ns = get_node_ids(c)
     return ((ns[1],), (ns[2],)) # f1, f2
 end
+function faces(::Type{RefLine})
```

Comment:
1) Should this "new" function have the same function name when it returns the local node indices? 
2) Is it possible to use this function to create the existing `faces(::AbstractCell)` functions? Seems bad to duplicate the numbering if it can be avoided without performance losses...

Reply:
Thanks for the review! I think the name makes sense for the input, however, `reference_faces` might be better  given the input is a reference shape. What do you think?